modal/app.py

```diff
@@ -39,6 +39,8 @@ class _App:
 
     _tag_to_object: Dict[str, _Handle]
     _tag_to_existing_id: Dict[str, str]
+    _object_id_to_object: Dict[str, _Handle]
```

Comment:
what's this needed for?

Reply:
Deserialization - used when looking up referenced objects that are inlined as part of the serialized function "closure" (having only an object_id in the pickled reference)